app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -92,8 +92,8 @@ def repo_discover
     param :label, String, :desc => N_("Organization label")
     param :url, String, :desc => N_("base url to perform repo discovery on")
     def cancel_repo_discover
-      # TODO: implement task canceling
-      render :json => { message: "not implemented" }
+      @organization.cancel_repo_discovery
+      render :nothing => true
```

Comment:
I feel like the user should be getting a task back or some information to indicate the result of requested action and whether it was successful or not.